### ISSUE TITLE
Seer crashes while being installed in ChessGUI due to wrong reaction to 'stop' command

### ISSUE THREAD

**Issue by rwbc** [connormcmonigle/seer-nnue#1]
Hi Connor,

While trying to reproduce an error for another user in talkchess, which had to do with the uci Weights option
having no default value (you fixed it in current dev I see), I found out about another bug.

Seer will always crash during installation in ChessGUI, because it reacts to 'stop' with a bestmove h1h1,
while not searching. (The GUI just wants to check the options and memorize the settings).
After this the GUI sends quit, but Seer disgracefully crashes.

http://talkchess.com/forum3/viewtopic.php?f=2&t=75433&start=100#p872858

I created a log with inbetween (I/O logger and more)
```
0.000: - Starting server: seer111-64.exe
0.031: - Priority class set to normal
0.047: - Thread priority set to normal
0.047: > uci
1.233: < id name Seer 1.11
1.233: < id author Connor McMonigle
1.233: < option name Clear Hash type button
1.233: < option name Threads type spin default 1 min 1 max 512
1.233: < option name Hash type spin default 128 min 1 max 65536
1.233: < option name Weights type string default save.bin
1.233: < uciok
1.373: > stop
1.373: < bestmove h1h1
1.373: > quit
1.373: < 
1.373: < This application has requested the Runtime to terminate it in an unusual way.
1.373: < Please contact the application's support team for more information.
3.370: - Client closed
3.370: - Stopping server communication
8.377: - Closing log
```

The uci specs say you should ignore stop, when you are not searching.

```
* if the engine receives a command which is not supposed to come, for example "stop" when the engine is
  not calculating, it should also just ignore it.

```

**Comment by connormcmonigle** [comment]
This is hopefully resolved in v1.2. Thanks Guenther!

**Comment by connormcmonigle** [comment]
Graham has confirmed Seer works under Chess GUI now.
